executor/executor.go

```diff
@@ -567,11 +563,21 @@ func (e *LimitExec) Open(ctx context.Context) error {
 	if err := e.baseExecutor.Open(ctx); err != nil {
 		return errors.Trace(err)
 	}
+	e.childResult = e.children[0].newChunk()
 	e.cursor = 0
 	e.meetFirstBatch = e.begin == 0
 	return nil
 }
 
+// Close implements the Executor Close interface.
+func (e *LimitExec) Close() error {
+	e.childResult = nil
+	if err := errors.Trace(e.baseExecutor.Close()); err != nil {
```

Comment:
You could just return it. It's ok not to have this if branch.

Reply:
This is what I mean in the previous comments.😂

Reply:
done 😂😂